docs/src/main/sphinx/security/authentication-types.rst

```diff
@@ -10,11 +10,16 @@ You can configure one or more authentication types with the
 ``http-server.authentication.type`` property. The following authentication types
 and authenticators are available:
 
-* ``PASSSWORD`` for  :doc:`password-file`, :doc:`ldap`, and :doc:`salesforce`
+* ``PASSWORD`` for:
```

Comment:
Using colon for a fragment like this seems wrong from a style perspective

Reply:
Colon introduces a list. Nevertheless, the other uses of "for" in the parent list don't have colons, so this one goes.